### DISCUSSION TITLE
Confused by "snapshots" after the upgrade to 0.14

### DISCUSSION THREAD

**Discussion by ovizii** [blakeblackshear/frigate#12873] [General]
The FAQ says that Snapshots have been deprecated in favor of the new review item format. And yet I still the see the snapshot icon in the UI? Also, the reference config file still shows the snapshots configuration.

![image](https://github.com/user-attachments/assets/f12e410a-437e-441c-b9ac-d06da20d5d8b)

**Comment by hawkeye217** [comment]
Snapshots have been deprecated in the review UI, but are still used for frigate+.

**Reply by ovizii** [reply]
Thanks for chiming in but as a home user withoput Frigate+ what will that button do for me? 
Also, can I then simply remove the snapshot section from my config file?

**Reply by NickM-27** [reply]
You can still use snapshots for notifications or something else. The button simply disables or enables snapshots the same as before. If you have no use for them outside of Frigate then yes it can be removed from the Frigate config

**Reply by ovizii** [reply]
Thanks, that totally makes sense, I forgot I had notifications sending me snapshots :-)

**Reply by Morphy99** [reply]
How are we supposed to go about object tuning without being able to see the snapshots with the object bounding box now they've been removed?

**Reply by hawkeye217** [reply]
Debugging review items will be returning in an upcoming version, which will include information about detected objects, including a bounding box. Currently, snapshots remain available on the filesystem, through the Frigate+ view, and through the Home Assistant media browser.

**Reply by ovizii** [reply]
I also just noticed that yesterday, thanks for pointing it out, I thought I was doing something wrong and merely fiddled around with the tuning option :-)
